### ISSUE TITLE
Russian translation

### ISSUE THREAD

**Issue by iegik** [dwmkerr/hacker-laws#240]
Why not in same repository?

Discussion in russian language: https://github.com/solarrust/hacker-laws/issues/32

**Comment by dwmkerr** [comment]
Yes in general it is probably easier to manage in the same repo. We're using GitLocalize to manage translations - this allows moderators to see changes as they come in and update them. There's a discussion on this in issue #234.
